.eslintrc

```diff
@@ -43,36 +50,35 @@ rules:
   prefer-spread: 2
   prefer-template: 2
 
-globals:
-  # Meteor globals
-  Meteor: false
-  DDP: false
-  Mongo: false
-  Session: false
-  Accounts: false
-  Template: false
-  Blaze: false
-  UI: false
-  Match: false
-  check: false
-  Tracker: false
-  Deps: false
-  ReactiveVar: false
-  EJSON: false
-  HTTP: false
-  Email: false
-  Assets: false
-  Handlebars: false
-  Package: false
-  App: false
-  Npm: false
-  Tinytest: false
-  Random: false
-  HTML: false
+  # eslint-plugin-meteor
+  meteor/globals: 2
+  meteor/no-zero-timeout: 2
+  meteor/no-session: 0
```

Comment:
There was a [proposal](https://meteor.hackpad.com/Proposal-Deprecate-Session-in-favor-of-ReactiveDict-0wbRKtE4GZ9) to deprecate usage of `Session`. In case any application followed this proposal, using `Session` can be explicitly forbidden with this rule.

The rule is disabled by default, so this line is redundant. All other rules are set to error out.